internal/tensorzero-node/src/database.rs

```diff
@@ -221,6 +222,23 @@ impl DatabaseClient {
 
         serde_json::to_string(&result).map_err(|e| napi::Error::from_reason(e.to_string()))
     }
+
+    #[napi]
+    pub async fn update_datapoints_metadata(&self, params: String) -> Result<String, napi::Error> {
```

Comment:
Why not take `update_datapoints_metadata(&self, dataset_name: String, request: String)` which is consistent with the backend?